entries/w/westernunion.com.json

```
@@ -1,16 +1,18 @@
 {
   "Western Union": {
     "domain": "westernunion.com",
-    "url": "https://www.westernunion.com/",
     "tfa": [
       "sms",
       "call",
       "custom-software"
     ],
+    "custom-software": [
+      "Authy"
+    ],
     "documentation": "https://business.westernunion.com/en-us/product-documents/tfa-overview/Set-up-two-Factor",
```

Western Union Business Solutions has been renamed Convera ([source](https://convera.com/blog/press-releases/western-union-business-solutions-to-be-renamed-convera/)). Any request to `business.westernunion.com/en-us` redirects to `convera.com`. I can't find any mention of 2FA on their site.

```suggestion
    "documentation": "https://business.westernunion.com/en-pl/product-documents/psd2-overview/Set-up-two-Factor",
```